Fixes issue #1642: Drop-down menu does not disappear

mheyen opened the pull request:
The drop-down-menu in the adminUI now disappears after launching „Start Task“ dialog. 
This also works for:
- The language menu, 
- if you select ? → Keyboard shortcuts, 
- and the other links in that menu.

Added control for click event in modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js 

This fixes #1642

mheyen committed:
Fixes issue #1642: Dropdown-Menu does not disappear

Menu does now disappear after launching „Start Task“ dialog. Added control for click event
in modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js

lkiesow commented on modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js:
```diff
@@ -28,9 +28,12 @@ angular.module('adminNg.directives')
     link: function ($scope, element) {
       element.on('click', function (event) {
+        var is_active = angular.element(this).hasClass('active');
         angular.element('[old-admin-ng-dropdown]').removeClass('active');
```
Instead of re-activating, it might be simpler to spare `this` from deactivation instead?
```suggestion
        angular.element('[old-admin-ng-dropdown]').not(this).removeClass('active');
```

lkiesow reviewed (changes requested):
Seems to work. It does break the tests though.
This might be an issue with the tests or a sign that something is wrong in some edge cases.
We need to make sure the tests run through.

I've also suggested an alternative which might be a bit easier though I don't have a strong opinion on this.

mheyen committed:
Update modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js

Co-authored-by: Lars Kiesow <lkiesow@uos.de>

mheyen committed:
Update oldDropdownDirective.js

in modules/admin-ui/src/main/webapp/scripts/shared/directives/oldDropdownDirective.js

mheyen committed:
Merge remote-tracking branch 'refs/remotes/origin/r/8.x' into r/8.x
